test/fixture/vcr_cassettes/repositories/contributors#list.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "request": {
+      "url": "https://api.github.com/repos/antonydenyer/tentatest/contributors"
+    },
+    "response": {
+      "body": "[{\"login\":\"octocat\",\"id\":1,\"avatar_url\":\"https://github.com/images/error/octocat_happy.gif\",\"gravatar_id\":\"\",\"url\":\"https://api.github.com/users/octocat\",\"html_url\":\"https://github.com/octocat\",\"followers_url\":\"https://api.github.com/users/octocat/followers\",\"following_url\":\"https://api.github.com/users/octocat/following{/other_user}\",\"gists_url\":\"https://api.github.com/users/octocat/gists{/gist_id}\",\"starred_url\":\"https://api.github.com/users/octocat/starred{/owner}{/repo}\",\"subscriptions_url\":\"https://api.github.com/users/octocat/subscriptions\",\"organizations_url\":\"https://api.github.com/users/octocat/orgs\",\"repos_url\":\"https://api.github.com/users/octocat/repos\",\"events_url\":\"https://api.github.com/users/octocat/events{/privacy}\",\"received_events_url\":\"https://api.github.com/users/octocat/received_events\",\"type\":\"User\",\"site_admin\":false,\"contributions\":32}]",
+      "status_code": 200,
+      "type": "ok"
+    }
+  }
+]
```

Comment:
Didn't see the value in having the full response, wanted to keep things close to what is being tested. Happy to add the full response in if you think it's of value.

Reply:
I think that's fine 👍